spring-cloud-task-stream/src/test/java/org/springframework/cloud/task/batch/listener/JobInstanceEventTests.java

```diff
@@ -36,4 +37,23 @@ public void testConstructor() {
 		assertEquals(INSTANCE_ID, jobInstanceEvent.getInstanceId());
 		assertEquals(JOB_NAME, jobInstanceEvent.getJobName());
 	}
+
+	@Test
+	public void testEmptyConstructor() {
+		JobInstanceEvent jobInstanceEvent = new JobInstanceEvent();
+		assertNull(jobInstanceEvent.getJobName());
+	}
+
+	@Test(expected = NullPointerException.class)
```

Comment:
I'm 99% sure we should never *expect* a NPE.  If this test passes, I feel like there is something wrong...